src/mono/wasm/Makefile

```diff
@@ -177,6 +177,9 @@ run-tests-%:
 run-browser-tests-%:
 	PATH="$(CHROMEDRIVER):$(PATH)" XHARNESS_COMMAND=test-browser $(DOTNET) build $(TOP)/src/libraries/$*/tests/ /t:Test $(_MSBUILD_WASM_BUILD_ARGS) $(MSBUILD_ARGS)
 
+run-browser-outerloop-tests-%:
+	PATH="$(CHROMEDRIVER):$(PATH)" XHARNESS_COMMAND=test-browser $(DOTNET) build $(TOP)/src/libraries/$*/tests/ /t:Test /p:OuterLoop=true $(_MSBUILD_WASM_BUILD_ARGS) $(MSBUILD_ARGS)
```

Comment:
We're getting an explosion of targets, should we just add a comment to the README.md to use `MSBUILD_ARG=/p:OuterLoop=true make run-browser-tests-<test>` ?

Reply:
Will update the README instead.